pandas/core/shared_docs.py

```diff
@@ -119,6 +119,8 @@
     Sort group keys. Get better performance by turning this off.
     Note this does not influence the order of observations within each
     group. Groupby preserves the order of rows within each group.
+    This argument has no effect on some aggregating functions, such as
+    :ref:`head, tail <basics.aggregate>` and ``nth[:]``.
```

Comment:
it might be good to mention "filtrations" rather than "some aggregating functions", and link to the "filtrations" section introduced in https://github.com/pandas-dev/pandas/pull/51704/files?

as yes, it might be good to note it in `as_index` too

Reply:
Thanks, I did as you suggested. I think ci failures aren't related to my changes.

Reply:
thanks!

not sure, it's showing
```
/home/runner/work/pandas/pandas/pandas/core/frame.py:docstring of pandas.core.frame.DataFrame.groupby:210: WARNING: Duplicate explicit target name: "groupby user guide".
/home/runner/work/pandas/pandas/pandas/core/frame.py:docstring of pandas.core.frame.DataFrame.groupby:210: WARNING: Duplicate explicit target name: "groupby user guide".
/home/runner/work/pandas/pandas/pandas/core/series.py:docstring of pandas.core.series.Series.groupby:188: WARNING: Duplicate explicit target name: "groupby user guide".
/home/runner/work/pandas/pandas/pandas/core/series.py:docstring of pandas.core.series.Series.groupby:188: WARNING: Duplicate explicit target name: "groupby user guide".
```
which might be due to the link you wrote?

Reply:
Thank you for the comment. I agree, the warnings are related to my changes. There are two different links in one doctsring with the same name, which causes the problem. I changed the name of my link. 

It’s interesting, while I run validation
`python scripts/validate_docstrings.py pandas.core.groupby.DataFrameGroupBy` and
`python scripts/validate_docstrings.py pandas.core.groupby.SeriesGroupBy` there were no warnings.

After validation, I cleaned and built new documentation locally. Then I just checked html, my links worked right. My mistake, I should check warnings raised by the build.